lib/trento/application/integration/discovery/policies/host_policy.ex

```diff
@@ -209,4 +302,91 @@ defmodule Trento.Integration.Discovery.HostPolicy do
   defp parse_storage_profile(%{data_disks: nil}), do: 0
   defp parse_storage_profile(%{data_disks: data_disks}), do: length(data_disks)
   defp parse_storage_profile(_), do: 0
+
+  # Saptune payload
+
+  defp format_saptune_payload_keys(map) when is_map(map) do
+    try do
+      for {key, val} <- map,
+          into: %{},
+          do: {snake_case(key), format_saptune_payload_keys(val)}
+    rescue
+      Protocol.UndefinedError -> map
+    end
+  end
+
+  defp format_saptune_payload_keys(list) when is_list(list) do
+    list
+    |> Enum.map(&format_saptune_payload_keys/1)
+  end
+
+  defp format_saptune_payload_keys(other_types), do: other_types
+
+  defp snake_case(payload) when is_binary(payload) do
+    payload |> String.replace(" ", "_") |> String.downcase()
+  end
+
+  defp format_saptune_services_list(services) when map_size(services) == 0, do: []
+
+  defp format_saptune_services_list(service_map) do
+    Enum.map(service_map, fn {service_name, status} ->
+      %{
+        "name" => service_name,
+        "enabled" => saptune_service_enabled?(status),
+        "active" => saptune_service_active?(status)
+      }
+    end)
+  end
+
+  defp saptune_service_enabled?(service_status) do
+    Enum.any?(service_status, fn status -> status == "enabled" end)
+  end
+
+  defp saptune_service_active?(service_status) do
+    Enum.any?(service_status, fn status -> status == "active" end)
+  end
+
+  defp format_enabled_solution([solution_enabled], [%{"note_list" => note_list}]) do
+    format_solution(%{
+      "solution_id" => solution_enabled,
+      "note_list" => note_list,
+      "applied_partially" => false
+    })
+  end
+
+  defp format_enabled_solution(_, _), do: []
+
+  defp format_applied_solution([solution_applied], [%{"note_list" => note_list}]) do
+    format_solution(Map.put(solution_applied, "note_list", note_list))
+  end
+
+  defp format_applied_solution(_, _), do: []
+
+  defp format_solution(%{
+         "solution_id" => solution_id,
+         "note_list" => note_list,
+         "applied_partially" => partially_applied
+       }),
+       do: %{"id" => solution_id, "notes" => note_list, "partial" => partially_applied}
+
+  defp format_saptune_staging_informations(%{
```

Comment:
Why not simply `format_saptune_staging` ?
In any case the final `informations` with a plural `s` looks wrong